### PR TITLE
doc updates to cluster management overview

### PR DESCRIPTION
**Pull request by csplinter**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/clustermgmt/cluster-overview.md [hunk 1]
```diff
@@ -4,17 +4,35 @@ linkTitle: "Overview"
...
...t, or workload clusters, you modify the EKS Anywhere Cluster specification, which is a Kubernetes Custom Resource for EKS Anywhere clusters. When you modify a field in an existing Cluster specification, EKS Anywhere reconciles the infrastructure and Kubernetes components until they match the new desired state you defined. These operations are asynchronous and you can validate the state of your cluster following the guidance on the [Verify Cluster page]({{< relref "observability/cluster-verify/" >}}). To change a cluster configuration, update the field(s) in the Cluster specification and apply it to your standalone or management cluster. To upgrade the Kubernetes version, update the k`ubernetesVersion` field in the Cluster specification and apply it to your standalone or management cluster. To scale the cluster, update the control plane, etcd, or worker node `count` in the Cluster specification and apply it to the standalone or management cluster, or alternatively use Cluster Autoscaler to automate the scaling process. This operational model facilitates a declarative, intent-based pattern for defining and managing Kubernetes clusters that aligns with modern IAC and GitOps practices.
```
Comment (jiayiwang7) [inline docs/content/en/docs/clustermgmt/cluster-overview.md:16]:
```suggestion
To perform cluster lifecycle operations for standalone, management, or workload clusters, you modify the EKS Anywhere Cluster specification, which is a Kubernetes Custom Resource for EKS Anywhere clusters. When you modify a field in an existing Cluster specification, EKS Anywhere reconciles the infrastructure and Kubernetes components until they match the new desired state you defined. These operations are asynchronous and you can validate the state of your cluster following the guidance on the [Verify Cluster page]({{< relref "observability/cluster-verify/" >}}). To change a cluster configuration, update the field(s) in the Cluster specification and apply it to your standalone or management cluster. To upgrade the Kubernetes version, update the `kubernetesVersion` field in the Cluster specification and apply it to your standalone or management cluster. To scale the cluster, update the control plane or worker node `count` in the Cluster specification and apply it to the standalone or management cluster, or alternatively use Cluster Autoscaler to automate the scaling process. This operational model facilitates a declarative, intent-based pattern for defining and managing Kubernetes clusters that aligns with modern IAC and GitOps practices.
```
external etcd count is immutable

File: docs/content/en/docs/clustermgmt/cluster-overview.md [hunk 2]
```diff
@@ -4,17 +4,35 @@ linkTitle: "Overview"
...
+
+### Security
+The Shared Responsibility Model for EKS Anywhere is different than other AWS services that run in AWS Cloud, and it is your responsibility for the ongoing security of EKS Anywhere clusters you operate. Be sure to review and follow the EKS Anywhere [Security Best Practices]({{< relref "security/best-practices/" >}}).
+
+### Support
+EKS Anywhere is open source and free to use at no cost. To receive support for your EKS Anywhere clusters, you can optionally purchase [EKS Anywhere Enterprise Subscriptions]({{< relref "../concepts/support-scope/">}}) to get 24/7 support from AWS subject matter experts and access to [EKS Anywhere Curated Packages]({{< relref "../concepts/packages/">}}). See the [License cluster page]({{< relref "support/cluster-license/">}})  for information on how to apply a license to your EKS Anywhere clusters, and the [Generate Support Bundle page]({{< relref "support/supportbundle/">}}) for details on how to create a support bundle that contains diagnostics that AWS uses to troubleshoot and resolve issues.
```
Comment (jiayiwang7) [inline docs/content/en/docs/clustermgmt/cluster-overview.md:28]:
```suggestion
EKS Anywhere is open source and free to use at no cost. To receive support for your EKS Anywhere clusters, you can optionally purchase [EKS Anywhere Enterprise Subscriptions]({{< relref "../concepts/support-scope/">}}) to get 24/7 support from AWS subject matter experts and access to [EKS Anywhere Curated Packages]({{< relref "../concepts/packages/">}}). See the [License cluster page]({{< relref "support/cluster-license/">}}) for information on how to apply a license to your EKS Anywhere clusters, and the [Generate Support Bundle page]({{< relref "support/supportbundle/">}}) for details on how to create a support bundle that contains diagnostics that AWS uses to troubleshoot and resolve issues.
```

**Review by jiayiwang7 — COMMENTED** [review]
this is great!

File: docs/content/en/docs/clustermgmt/cluster-overview.md [hunk 3]
```diff
@@ -4,17 +4,35 @@ linkTitle: "Overview"
...
 
-* Directly in Kubernetes itself (such as adding nodes with `kubectl`)
-* Through the EKS Anywhere API (such as deleting a cluster with `eksctl`).
-* Through tools which interface with the Kubernetes API (such as [managing a cluster with `terraform`]({{< relref "./cluster-terraform" >}}))
+### Cluster Lifecycle Operations
+
+The tools available for cluster lifecycle operations (create, update, upgrade, scale, delete) vary based on the EKS Anywhere architecture you run. You must use the eksctl CLI for cluster lifecycle operations with standalone clusters and management clusters. If you are running a management / workload cluster architecture, you can use the management cluster to manage one to many downstream workload clusters. With the management cluster architecture, you can use the eksctl CLI, any Kubernetes API-compatible client, or Infrastructure as Code (IAC) tooling such as Terraform and GitOps to manage the lifecycle of workload clusters. For details on the differences between the architecture options, reference the [Architecture page]({{< relref "../concepts/architecture/" >}}).
```
[the remark this answers is not in the document]
Comment (csplinter) [reply docs/content/en/docs/clustermgmt/cluster-overview.md:14]: I think so? but we are currently distinguishing the two on the [Architecture page](https://anywhere.eks.amazonaws.com/docs/concepts/architecture/).

File: docs/content/en/docs/clustermgmt/cluster-overview.md (hunk 1 above)
Comment (csplinter) [reply docs/content/en/docs/clustermgmt/cluster-overview.md:16]: Really? what happens if you need to scale etcd after cluster creation?

Comment (jiayiwang7) [reply docs/content/en/docs/clustermgmt/cluster-overview.md:16]: can't do that..

File: docs/content/en/docs/clustermgmt/cluster-overview.md [hunk 4]
```diff
@@ -4,17 +4,35 @@ linkTitle: "Overview"
...
+
+You additionally must configure CIDR blocks for Pods and Services in the Cluster specification during cluster creation. Each node in EKS Anywhere clusters receives an IP range subset from the Pod CIDR block, see (Node IPs configuration)({{< relref "../getting-started/optional/cni/#node-ips-configuration-option" >}}) for more details. EKS Anywhere uses Cilium as the Container Networking Interface (CNI). The Cilium version in EKS Anywhere, "EKS Anywhere Cilium", contains a subset of the open source Cilium capabilities. EKS Anywhere configures Cilium in [Kubernetes host-scope IPAM mode](https://docs.cilium.io/en/v1.12/concepts/networking/ipam/kubernetes/), which delegates the address allocation to each individual node in the cluster. You can optionally replace the EKS Anywhere Cilium installation with a different CNI after cluster creation, reference the [Custom CNI page]({{< relref "networking/cluster-replace-cilium" >}}) for details.
```
Comment (chrisnegus) [inline docs/content/en/docs/clustermgmt/cluster-overview.md:22]:
```suggestion
You additionally must configure CIDR blocks for Pods and Services in the Cluster specification during cluster creation. Each node in EKS Anywhere clusters receives an IP range subset from the Pod CIDR block, see [Node IPs configuration]({{< relref "../getting-started/optional/cni/#node-ips-configuration-option" >}}) for more details. EKS Anywhere uses Cilium as the Container Networking Interface (CNI). The Cilium version in EKS Anywhere, "EKS Anywhere Cilium", contains a subset of the open source Cilium capabilities. EKS Anywhere configures Cilium in [Kubernetes host-scope IPAM mode](https://docs.cilium.io/en/v1.12/concepts/networking/ipam/kubernetes/), which delegates the address allocation to each individual node in the cluster. You can optionally replace the EKS Anywhere Cilium installation with a different CNI after cluster creation. Reference the [Custom CNI page]({{< relref "networking/cluster-replace-cilium" >}}) for details. 
```

**Review by chrisnegus — COMMENTED** [review]
I think this is a good collection and level of information for an overview. I'm going to suggest that we provide this kind of information for overview pages in other sections (such as those currently being worked on for the providers).

**Pull request merged**